test/lib/ceych.js

```diff
@@ -137,6 +137,107 @@ describe('ceych', () => {
         });
       });
     });
+
+    it('passes the parameters to the wrapped function', (done) => {
+      const myFun = (one, two, three, cb) => {
+        cb(null, one + two + three);
+      };
+      const wrappedFn = ceych.wrap(myFun);
+
+      wrappedFn(1, 2, 3, (err, results) => {
+        assert.ifError(err);
+        assert.strictEqual(results, 6);
+
+        done();
+      });
+    });
```

Comment:
You actually didnt have a test to ensure the returned function passed it's arguments onto the wrapped one - that caught me out!